contracts/UFragmentsPolicy.sol

```diff
@@ -57,12 +55,15 @@ contract UFragmentsPolicy is Ownable {
     // The number of rebase cycles since inception
     uint256 public epoch;
 
-    uint256 private constant RATE_DECIMALS = 18;
+    // TODO: Explain this parameter
+    uint256 public targetRate;
 
-    uint256 private constant TARGET_RATE = 1 * 10**RATE_DECIMALS;
+    uint256 private constant CPI_DECIMALS = 6;
 
-    int256 private constant TARGET_RATE_SIGNED = int256(TARGET_RATE);
+    uint256 private constant TARGET_CPI = 100 * 10**CPI_DECIMALS;
+    uint256 private constant INITIAL_CPI = 251 * 10**CPI_DECIMALS;
```

Comment:
`INITIAL_CPI` is the CPI at the time of launch. It's used to calculate the `targetRate` at the time of launch. 

```
targetRate = INITIAL_CPI / TARGET_CPI
```

Another way to do this to hard code the `INITIAL_TARGET_RATE`.